internal/test_helpers/fixtures/bash/history_persistence/pass

```diff
@@ -0,0 +1,162 @@
+[33m[your-program] [0m$ history -a /tmp/grape_shell_history_test
+[33m[stage-3] [0m[92m✓ history -a command executed[0m
+[33m[stage-3] [0m[94mHistory file content after second append:[0m
+[33m[stage-3] [0m[grape_shell_history_test] echo strawberry pineapple blueberry
+[33m[stage-3] [0m[grape_shell_history_test] echo orange pineapple apple
+[33m[stage-3] [0m[grape_shell_history_test] echo pineapple apple mango
+[33m[stage-3] [0m[grape_shell_history_test] echo pineapple banana pear
+[33m[stage-3] [0m[grape_shell_history_test] echo grape pear
+[33m[stage-3] [0m[grape_shell_history_test] history
+[33m[stage-3] [0m[grape_shell_history_test] history -a /tmp/grape_shell_history_test
+[33m[stage-3] [0m[grape_shell_history_test] history -a /tmp/grape_shell_history_test
+[33m[stage-3] [0m[92m✓ Command "echo grape pear" count preserved (1)[0m
+[33m[stage-3] [0m[92m✓ Command "echo orange pineapple apple" count preserved (1)[0m
+[33m[stage-3] [0m[92m✓ Command "echo pineapple apple mango" count preserved (1)[0m
+[33m[stage-3] [0m[92m✓ Command "echo pineapple banana pear" count preserved (1)[0m
+[33m[stage-3] [0m[92m✓ Command "echo strawberry pineapple blueberry" count preserved (1)[0m
+[33m[your-program] [0m$ 
+[33m[stage-3] [0m[92mTest passed.[0m
+
+[33m[stage-2] [0m[94mRunning tests for Stage #2: kz7[0m
+[33m[stage-2] [0m[94mRunning ./your_shell.sh[0m
+[33m[your-program] [0m$ echo mango banana apple
+[33m[your-program] [0mmango banana apple
+[33m[stage-2] [0m[92m✓ Ran echo mango banana apple[0m
+[33m[your-program] [0m$ echo strawberry pear
+[33m[your-program] [0mstrawberry pear
+[33m[stage-2] [0m[92m✓ Ran echo strawberry pear[0m
+[33m[your-program] [0m$ history
+[33m[your-program] [0m    1  echo mango banana apple
+[33m[your-program] [0m    2  echo strawberry pear
+[33m[your-program] [0m    3  history
+[33m[stage-2] [0m[92m✓ History before exiting is correct[0m
+[33m[your-program] [0m$ exit 0
+[33m[your-program] [0mexit
+[33m[stage-2] [0m[94mHistory file content:[0m
+[33m[stage-2] [0m[strawberry_shell_history_test] echo mango banana apple
+[33m[stage-2] [0m[strawberry_shell_history_test] echo strawberry pear
+[33m[stage-2] [0m[strawberry_shell_history_test] history
+[33m[stage-2] [0m[strawberry_shell_history_test] exit 0
+[33m[stage-2] [0m[92m✓ History file has correct number of lines[0m
+[33m[stage-2] [0m[92m✓ Found command "echo mango banana apple" in history file[0m
+[33m[stage-2] [0m[92m✓ Found command "echo strawberry pear" in history file[0m
+[33m[stage-2] [0m[92m✓ Found command "history" in history file[0m
+[33m[stage-2] [0m[92m✓ Found command "exit 0" in history file[0m
+[33m[stage-2] [0m[92mTest passed.[0m
+
+[33m[stage-1] [0m[94mRunning tests for Stage #1: zp4[0m
+[33m[stage-1] [0m[94mRunning ./your_shell.sh[0m
+[33m[stage-1] [0m[94mHistory file content:[0m
```

Comment:
We aren't mentioning the path of the file anywhere, so it could be confusing that `mango_shell_history_test` means. Also, looks like both the prefix and the text are the same color, that's not what the interpreter challenge does - the prefix is yellow (look at `updateSecondaryPrefix` on logger). 

I'd recommend an alternate format a bit like this: 

```
[stage-5] Writing contents to history file (`/tmp/mango.txt`)
[stage-5] [/tmp/mango.txt] echo 1
[stage-5] [/tmp/mango.txt] echo 2
[stage-5] $ history
```

(The `.txt` makes it more clear that we're dealing with a file)